CHANGELOG.md

```diff
@@ -22,3 +22,8 @@
  * When blackboxing the SDK it doesn't stop on Dart exceptions, since they're in
    blackboxed source. Set the blackbox by source range to get around this. This
    isn't visible in the UI, so it's hard-coded.
+ * Adding dart links to runtime types for objects
+ * Fixed a bug resulting in links not being generated whenever the first entry
```

Comment:
What does this mean. You hovered over the thing that was the very first entry in the sourcemap? Or something else?

Reply:
More specifically, there was code that looked something like "index ? doStuff : return undefined". This resulted in bad links whenever the very first breakpoint was the breakpoint to be linkified.